src/modules/services/components/FilePicker/index.spec.jsx

```
@@ -279,6 +279,30 @@ describe('FilePicker', () => {
+    fireEvent.click(getByTestId('public-link-btn'))
+
+    await findByTestId('link-access-modal')
+    fireEvent.click(getByTestId('confirm-link-access-btn'))
+    fireEvent.click(getByTestId('confirm-link-access-btn'))
+
+    expect(mockOnChange).toHaveBeenCalledTimes(1)
+
+    resolveFirst(null)
+    await waitFor(() =>
+      expect(mockShowAlert).not.toHaveBeenCalled()
+    )
```

ESLint

[error] Replace `⏎······expect(mockShowAlert).not.toHaveBeenCalled()⏎····` with `·expect(mockShowAlert).not.toHaveBeenCalled()`

(prettier/prettier)